selftests/unit/test_vt_init.py

```diff
@@ -0,0 +1,38 @@
+import importlib
+import os
+import tempfile
+import unittest
+
+from selftests import BASEDIR
+
+
+class VtInitTest(unittest.TestCase):
+
+    @staticmethod
+    def _swap():
+        data_dir = os.path.join(BASEDIR, "selftests", ".data")
+        config_file = os.path.join(BASEDIR, "avocado_vt", "conf.d", "vt.conf")
+        config_test_file = os.path.join(data_dir, "vt_test.conf")
+        tmp, tmp_name = tempfile.mkstemp(suffix=".conf", dir=data_dir)
+        os.rename(config_test_file, tmp_name)
+        os.rename(config_file, config_test_file)
+        os.rename(tmp_name, config_file)
+
+    def setUp(self):
+        self._swap()
+
+    def test_something(self):
```

Comment:
Perhaps a better name would be `test_early_settings` or something of the like.

Reply:
A minimal docstring might also be useful.

Reply:
Oh, of course, sorry I totally forget to rename that :D